lib/swagger-mock-validator/validate-spec-and-mock/validate-parsed-mock-request-body.ts

```diff
@@ -30,13 +44,8 @@ const specAndMockHaveNoBody = (parsedMockInteraction: ParsedMockInteraction,
                                parsedSpecOperation: ParsedSpecOperation) =>
     !parsedSpecOperation.requestBodyParameter && !parsedMockInteraction.requestBody.value;
 
-const isNotSupportedMediaType = (parsedSpecOperation: ParsedSpecOperation) =>
-    parsedSpecOperation.consumes.value.length > 0 &&
-    !isMediaTypeSupported('application/json', parsedSpecOperation.consumes.value);
-
 const shouldSkipValidation = (parsedMockInteraction: ParsedMockInteraction,
                               parsedSpecOperation: ParsedSpecOperation) =>
-    isNotSupportedMediaType(parsedSpecOperation) ||
```

Comment:
This is possibly a breaking change. Previously, when the request mediaType is unrecognised, no validation is performed at all. This doesn't seem right, so I'm removing this short-circuit.